index.js

```diff
@@ -685,49 +689,63 @@ async function main() {
       }
 
       // disable cdk prompting if user included `--yes` flag
-      if (cmd.yes && (cdkArgs.includes('deploy') || cdkArgs.includes('changeset'))) {
+      if (
+        cmd.yes &&
+        (cdkArgs.includes('deploy') || cdkArgs.includes('changeset'))
+      ) {
         cdkArgs.push('--require-approval-never');
       }
 
-      if (['deploy', 'destroy', 'changeset'].some((c) => cdkArgs.includes(c))) {
+      if (
+        ['deploy', 'destroy', 'changeset'].some((c) => {
+          return cdkArgs.includes(c);
+        })
+      ) {
         // check that we're not using a wildly different version of the cli
         if (stackExists && !cmd.yes && !(await cmd.stackVersionDiffCheck())) {
           exitWithSuccess();
         }
         // production failsafe if we're actually changing anything
-        if (!await confirmProductionOp(cmd.yes)) {
+        if (!(await confirmProductionOp(cmd.yes))) {
           exitWithSuccess();
         }
       }
 
       // Set some default removal policy options depending on if this is a "prod" account
-      if (cdkStackProps.deployConfig.dbOptions && !cdkStackProps.deployConfig.dbOptions.removalPolicy) {
-        cdkStackProps.deployConfig.dbOptions.removalPolicy = await isProdAccount() ? 'RETAIN' : 'DESTROY';
+      if (
+        cdkStackProps.deployConfig.dbOptions &&
+        !cdkStackProps.deployConfig.dbOptions.removalPolicy
+      ) {
+        cdkStackProps.deployConfig.dbOptions.removalPolicy =
+          (await isProdAccount()) ? 'RETAIN' : 'DESTROY';
       }
 
       /**
        * Write out the stack properties to a temp json file for
        * the CDK subprocess to pick up
        */
-      tempy.write.task(JSON.stringify(cdkStackProps), async (tempPath) => {
-        // tell the CDK subprocess where to find the stack properties file
-        envAdditions.CDK_STACK_PROPS_FILE_PATH = tempPath;
-
-        const execOpts = {
-          stdio: 'inherit',
-          // exec the cdk process in the cdk directory
-          cwd: __dirname, // path.join(__dirname, 'cdk'),
-          // inject our additional env vars
-          env: { ...process.env, ...envAdditions },
-        };
-
-        try {
-          execSync(['node_modules/.bin/cdk', ...cdkArgs].join(' '), execOpts);
-          exitWithSuccess('done!');
-        } catch (err) {
-          exitWithError(err.msg);
-        }
-      });
+      await tempy.write.task(
+        JSON.stringify(cdkStackProps),
```

Comment:
I'm not sure if this file needs to be human-readable, but if it should be readable, I like `JSON.stringify(cdkStackProps, null, 2)`